ui/src/components/HelpAccordian.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from "react";
+import styled from "styled-components";
+import { colors } from "../styles/colors";
+import Accordion from "@material-ui/core/Accordion";
+import AccordionSummary from "@material-ui/core/AccordionSummary";
+import AccordionDetails from "@material-ui/core/AccordionDetails";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import { Body1 } from "../styles/typography";
+
+const StyledAccordianSummary = styled(AccordionSummary)`
+  flex-direction: row-reverse;
+`;
+
+const StyledBox = styled.div`
+  background: ${colors.neutral};
+  border-radius: 10px;
+  padding: 24px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledAccordian = styled(Accordion)`
+  && {
+    box-shadow: none;
+  }
+
+  .MuiAccordionSummary-root {
+    padding: 0px;
+  }
+
+  .MuiAccordionSummary-content {
+    margin: 10px 0px 10px 20px;
+  }
```

Comment:
```suggestion
  &:hover {
      color: ${colors.primaryDark1};
    }
  }
```
![image](https://user-images.githubusercontent.com/19617248/101398795-a3daa100-389c-11eb-8cc0-e1bad4470d16.png)

Reply:
Didn't comment it but also for the dropdown arrow icon ^^